app/views/base/index.html.erb

```diff
@@ -74,20 +72,12 @@
     </div>
   </div>
 </section>
-<section class="schools">
-  <h2>
-    <%= t 'landing.used-by.header' %>
-  </h2>
-  <% t('landing.used-by.schools').each do |school| %>
-    <%= image_tag school[:image], title: school[:name] %>
-  <% end %>
-</section>
 <section class="stinger">
   <%= image_tag 'logo.png', title: t('header.brand') %>
   <h1>
     <%= t 'landing.stinger.pitch' %>
   </h1>
-  <a class="call-to-action">
+  <a class="call-to-action" href="<%= new_user_path %>">
     <%= t 'landing.stinger.call-to-action' %>
   </a>
```

Comment:
<%= link_to  t('landing.stinger.call-to-action'), new_user_path, class: "call-to-action" %>